edalize/xcelium.py

```diff
@@ -76,55 +71,55 @@ def get_doc(cls, api_ver):
         if api_ver == 0:
             return {'description' : "Xcelium simulator from Cadence Design Systems",
                     'lists' : [
-                        {'name' : 'xmvhdl_options',
+                        {'name' : 'elab_options',
                          'type' : 'String',
-                         'desc' : 'Additional options for compilation with xmvhdl'},
-                        {'name' : 'xmvlog_options',
+                         'desc' : 'Additional run options for xrun -elaboration'},
+                        {'name' : 'xrun_prefix',
                          'type' : 'String',
-                         'desc' : 'Additional options for compilation with xmvlog'},
-                        {'name' : 'xmsim_options',
+                         'desc' : 'prefix for using submission servers like bjob or nc'},
```

Comment:
Drop this and use the EDALIZE_LAUNCHER environment variable instead. There is an upcoming refactoring that, among other things, will standardize this for all the backend flows

Reply:
I will look into it